CHANGELOG.md

```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
 
 | Class/Property/Method/Event | Alternate option | Version deprecated |
 |----------|-------------|--------------------|
+| `BasemapLayerList.statusIndicatorsVisible` | `BasemapLayerList.visibleElements.statusIndicators` | 4.15 |
 | `Bookmarks.bookmarkCreationOptions` | `Bookmarks.defaultCreateOptions` | 4.18 |
+| `esri/tasks/ImageServiceIdentifyTask` | `imageService.identify` | 4.18 |
+| `esri/tasks/support/ImageServiceIdentifyResult` | `ImageIdentifyResult` | 4.18 |
+| `esri/tasks/support/ImageServiceIdentifyParameters` | `ImageIdentifyParameters` | 4.18 |
+| `FeatureForm.description` and `FeatureFormViewModel.description`  | `FormTemplate.description` set either on the `FeatureForm` or `FeatureLayer.formTemplate` | 4.18 |
+| `FeatureForm.fieldConfig` and `FeatureFormViewModel.fieldConfig` | `FieldElement/GroupElement` set in the `FormTemplate.elements` | 4.16 |
+| `FeatureForm.title` and `FeatureFormViewModel.title` | `FormTemplate.title` set either on the `FeatureForm` or `FeatureLayer.formTemplate`  | 4.18 |
 
 Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/latest/breaking-changes/) guide topic for a complete list of breaking changes across all releases of the 4x API.
 
 ## Bug fixes and enhancements
 
-- BUG-000141284: Fixed an issue with [WMSLayer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-WMSLayer.html) where custom parameters would be added to the GetLegendGraphic request with a `?` instead of an `&`, causing the request to fail.
-- Fixed an issue where layer effects were not being applied to ImageryLayer.
-- Fixed an issue where VectorFieldRenderer would not update when used with the TimeSlider widget.
+* TBD
```

Comment:
I added two bug fixes to the release notes this morning. not sure when `/next` was published but they might be able to be included here

Reply:
np, I can copy them over.

Reply:
Done.